lib/std/zig/system/NativePaths.zig

```diff
@@ -86,6 +86,7 @@ pub fn detect(arena: Allocator, native_target: std.Target) !NativePaths {
         if (std.zig.system.darwin.isSdkInstalled(arena)) sdk: {
             const sdk = std.zig.system.darwin.getSdk(arena, native_target) orelse break :sdk;
             try self.addLibDir(try std.fs.path.join(arena, &.{ sdk, "usr/lib" }));
+            try self.addFrameworkDir(try std.fs.path.join(arena, &.{ sdk, "System/Library/PrivateFrameworks" }));
```

Comment:
I don't think it's a good idea to include PrivateFrameworks by default.

Reply:
Could you explain why? I think having the build system look first for frameworks and then for private frameworks under the native SDK path is a good default.

Reply:
The private framework is not meant to be used by developers, if developers want to use it anyway, they need to set the required paths all by themselves.

For example clang don't include the private framework path by default.

Reply:
Private frameworks can be used by developers. They are called private because App Store apps aren't allowed to use them, only Apple apps can. See: https://theapplewiki.com/wiki/Filesystem:/System/Library/Frameworks

> Public frameworks are allowed to be used in App Store apps. Private frameworks are intended to be used only by Apple's apps, and are more unstable against firmware changes, but many of the interesting features are in the private frameworks.

Reply:
But I see how this might seem like a workaround for the fact that `zig` is unable to find frameworks under root, while `clang` seems to be able to.